frontend/pages/dashboard.vue

```diff
@@ -1,25 +1,238 @@
+        icon: 'pi pi-sort-alpha-down',
+        command: () => {
+          sortJourneys('destination-desc');
+        }
+      },
+    ]
+  },
+
+]);
+
+const toggle = (event: Event) => {
+  menu.value.toggle(event);
+};
+
+
+async function fetchJourneys() {
+  const client = useSanctumClient();
+  await client("/api/journey", {
+    method: "GET",
+    async onResponse({ response }) {
+      journeys.value = response._data;
+      currentJourneys.value = response._data;
+      store.setJourneys(response._data);
+    }
+  });
+
+}
+
+
+async function searchJourneys() {
+  const data: Journey[] = journeys.value;
+  const results: Journey[] = data.filter((obj: Journey) => {
+    return obj.name.toLowerCase().includes(searchValue.value.toLowerCase()) ||
+      obj.destination.toLowerCase().includes(searchValue.value.toLowerCase());
+  });
+
+  currentJourneys.value = results;
+}
+
+function sortJourneys(sortKey: String) {
+  currentJourneys.value.sort((a: Journey, b: Journey) => {
+    if (sortKey === 'name-asc') return b.name.localeCompare(a.name);
+    if (sortKey === 'name-desc') return a.name.localeCompare(b.name);
+    if (sortKey === 'startdate-asc') return a.from.localeCompare(b.from);
+    if (sortKey === 'startdate-desc') return b.from.localeCompare(a.from);
+    if (sortKey === 'destination-asc') return b.destination.localeCompare(a.destination);
+    if (sortKey === 'destination-desc') return a.destination.localeCompare(b.destination);
+    return 0;
+  });
+}
+
+fetchJourneys();
+
 </script>
 
 <template>
-  <div class="font-nunito bg-surface flex justify-between">
-    <h1 class="mt-10 ml-10 h-20 font-bold text-4xl">
-      <T keyName="common.dashboard" />
-    </h1>
-    <NuxtLink to="/journey/new" class="mt-10">
-      <button class="bg-cta-bg border-2 border-cta-border text-text min-w-44 py-2 rounded-lg font-bold mr-10">
-        <T keyName="form.header.journey.create" />
-      </button>
-    </NuxtLink>
+  <div class="font-nunito px-2 md:px-8 lg:px-20 text-text dark:text-white">
+    <div id="header" class="border-b-2 border-border mt-5 pb-3 md:pb-5 flex justify-between items-center">
+      <div class="flex flex-row items-center">
+        <SvgDashboardIcon class="md:w-9 md:h-9 mr-1" />
+        <h1 class="text-3xl md:text-5xl font-medium mt-1">
+          <T keyName="common.dashboard" />
+        </h1>
+      </div>
+      <div id="right-header" class="flex flex-row items-center">
+        <div id="search-and-filter" class="hidden lg:flex flex-row border-r-2 mr-4 border-border-grey">
+          <div id="search" class="relative mr-2.5" v-tooltip.top="t('dashboard.tooltip.search')">
+            <input type="text" ref="searchInput" @input="searchJourneys" v-model="searchValue"
+              class="rounded-3xl bg-input dark:bg-input-dark placeholder-input-placeholder dark:placeholder-text-light-dark border px-3 py-1.5 border-border-grey dark:border-input-dark focus:outline-none focus:ring-1 focus:ring-cta-border "
```

Comment:
In der Handy version wäre es mir lieber wenn das search feld vllt etwas länger gezogen wird 
![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/125883002/6bbff62b-f94c-469b-9244-7013bc8a68db)

Reply:
und wenn es geht bitte den "Neue Reise" Button mit dem feld drunter allignen

Reply:
Can't reproduce second comment.